Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -399,7 +430,7 @@ std::vector<u8> TicketReader::GetRawTicketView(u32 ticket_num) const
 
   // Copy the ticket version to the buffer (a single byte extended to 4).
```

Comment:
Just checked my ES IDB and it looks like this is a mistake; the ticket version in the ticket view is a single byte (u8), not a u32. Could you fix the type of `TicketView::version` and then just do `view[0] = m_bytes[offsetof(Ticket, version)];`?

(Maybe add a TicketReader function to get the ticket version?)